### ISSUE TITLE
[Feature Request] Add progress bar or progress information on notification

### ISSUE THREAD

**Issue by jdvr** [flackbash/AudioAnchor#63]
While listening to a relatively long podcast some times I want to check how much time is left.

It would be much more agile to be able to check it directly in the notification without having to open the app.

I can draft a PR if you are agree in add this feature.

**Comment by flackbash** [comment]
Yes, I agree. You are very welcome to implement this if you like :)

I would however prefer to have the progress bar uneditable or at least have an option to have it uneditable such that one cannot by accident jump to a different position in the file.

**Comment by jdvr** [comment]
Ok. I'll try to draft something during the weekend.
